ludwig/schema/features/image_feature.py

```diff
@@ -23,6 +28,21 @@ class ImageInputFeatureConfigMixin(BaseMarshmallowConfig):
         default="stacked_cnn",
     )
 
+    augmentation: Union[bool, List[BaseAugmentationConfig]] = schema_utils.OneOfOptionsField(
```

Comment:
As mentioned in an above comment, without explicitly setting `allow_none=False` here the parameter actually can be set to `None`. If that's intentional then also add it to the type annotation.

Reply:
Since the intent is not to support `None`, I'll leave this alone and look for guidance on the earlier comment.

Reply:
To fix this, simply add `None` to the `Union` and set `allow_none=False`

Reply:
done.